tutorials/plot_brainstorm_auditory.py

```diff
@@ -0,0 +1,356 @@
+# Here we compute the saccade and EOG projectors for magnetometers and add
+# them to the raw data. The projectors are added to both runs.
+saccade_epochs = mne.Epochs(raw2, saccades, 1, 0., 0.5, preload=True,
+                            segment_reject=False)
+
+projs_saccade = mne.compute_proj_epochs(saccade_epochs, n_mag=1, n_eeg=0,
+                                        desc_prefix='saccade')
+if use_precomputed:
+    proj_fname = op.join(data_path, 'MEG', 'bst_auditory',
+                         'bst_auditory-eog-proj.fif')
+    projs_eog = mne.read_proj(proj_fname)[0]
+else:
+    projs_eog, _ = mne.preprocessing.compute_proj_eog(raw2.load_data(),
+                                                      n_mag=1, n_eeg=0)
+for raw in raws[:2]:
+    raw.add_proj(projs_saccade)
+    raw.add_proj(projs_eog)
+del saccade_epochs  # To save memory
+
+##############################################################################
+# Visually inspect the effects of projections. Click on 'proj' button at the
+# bottom right corner to toggle the projectors on/off. EOG events can be
+# plotted by adding the event list as a keyword argument. As the bad segments
+# and saccades were added as annotations to the raw data, they are plotted as
+# well.
+raw1.plot()
+raw2.plot(block=True)
+
+##############################################################################
+# Typical preprocessing step is the removal of power line artifact (50 Hz or
+# 60 Hz). Here we notch filter the data at 60, 120 and 180 to remove the
+# original 60 Hz artifact and the harmonics. The power spectra are plotted
+# before and after the filtering to show the effect. The drop after 600 Hz
+# appears because the data was filtered during the acquisition. To save time
+# and memory we do the filtering at evoked stage, which is not something you
+# usually would do.
+if not use_precomputed:
+    meg_picks = mne.pick_types(raw.info, meg=True, eeg=False)
+    raws[0].plot_psd(picks=meg_picks)
+    notches = np.arange(60, 181, 60)
+    for raw in raws:
+        raw.notch_filter(notches)
+    raws[0].plot_psd(picks=meg_picks)
+
+##############################################################################
+# We also lowpass filter the data at 100 Hz to remove the hf components.
+if not use_precomputed:
+    for raw in raws:
+        raw.filter(None, 100.)
+
+##############################################################################
+# Epoching and averaging.
+# First some parameters are defined and events extracted from the stimulus
+# channel (UPPT001). The rejection thresholds are defined as peak-to-peak
+# values and are in T / m for gradiometers, T for magnetometers and
+# V for EOG and EEG channels.
+tmin, tmax = -0.1, 0.5
+event_id = dict(standard=1, deviant=2)
+reject = dict(mag=4e-12, eog=250e-6)
+# find events
+events1 = mne.find_events(raw1, stim_channel='UPPT001')
+events2 = mne.find_events(raw2, stim_channel='UPPT001')
```

Comment:
I suspect that is old -- previously we didn't have `preload=False` mode for BTi, and we probably didn't even have CTF when this was written. It would be a lot nicer (and probably save memory) not to have these duplications. Do you have time to look into this issue? We could try to fix that, then rebase this so it's cleaner and lower memory.